src/chef-mover/_checkouts/oc_erchef/rebar.config

```diff
@@ -15,8 +15,8 @@
   {git,"git://github.com/opscode/chef_authn.git",
           "5d0577f693404898d6f05dbcbb38ffe2cc36c15d"}},
  {couchbeam,".*",
-  {git,"git://github.com/opscode/couchbeam.git",
-         {branch, "lbaker/fixes_for_erlang19"}}},
+  {git,"git://github.com/chef/couchbeam.git",
```

Comment:
opscode and chef both redirect to the same repo in chef-boneyard (https://github.com/chef-boneyard/couchbeam)

Reply:
Yes.  I believe Mark said I should use the 'chef' name for all of those just to be consistent, so I changed them.